lib/home_widget.dart

```diff
@@ -50,6 +50,27 @@ class HomeWidget {
     });
   }
 
+  /// Pins the HomeScreen Widget
```

Comment:
Add docs when result `true` is returned, e.g only when widget is successfully pinned